OasisDEX/oasis-borrow
# new share icon

# [New Share icon](https://app.shortcut.com/oazo-apps/story/7784/update-the-share-vaults-icon)
  
## Changes 👷‍♀️
- Changed share icon, also corrected some CSS details to make it fit 
  
## How to test 🧪
- Check on discovery page 
![image](https://user-images.githubusercontent.com/6871923/215784620-c0185b75-fe2b-4aa5-8e46-570145846e44.png)

- Check on VaultOverview page 
![image](https://user-images.githubusercontent.com/6871923/215784502-fb53c582-874f-4b04-a59a-0ad4325af1ff.png)

- Also check AAVE, GUNI, DSR
![image](https://user-images.githubusercontent.com/6871923/215784850-c476ea32-d297-434e-b142-e49e9521e251.png)
